src/instructlab/configuration.py

```diff
@@ -1520,6 +1546,15 @@ def map_train_to_library(ctx, params):
     train_args = TrainingArgs(**params)
     torch_args = TorchrunArgs(**params)
 
+    # let's handle the case when `--model-id` was provided
+    if params["model_id"]:
```

Comment:
I know that this will work for now (I think I wrote it this way originally lol), but we might want to change this to use `params.get` here in case a refactor happens down the road so we can avoid a `KeyError` randomly becoming a bug:

```suggestion
    if params.get("model_id"):
```